test/app/.babelrc

```diff
@@ -1,13 +1,10 @@
 {
   "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "targets": {
-          "esmodules": true
-        }
-      }
-    ],
-    "@babel/preset-typescript"
+    "@babel/preset-typescript",
+    "@babel/preset-env"
+  ],
+  "plugins": [
+    "@babel/plugin-transform-runtime",
```

Comment:
Can we convert this file to `babel.config.js` and add the `transform-runtime` plugin conditionally based on an environment variable? This plugin makes debugging very difficult, it is much better for our DevEx if we can exclude it by default and only include it where needed (like IE11)